apps/dep-graph-client/src/app/graph.ts

```diff
@@ -0,0 +1,384 @@
+            'text-halign': 'center',
+            'text-valign': 'center',
+            'padding-left': '16px',
+          },
+        },
+        {
+          selector: 'node[type="lib"]',
+          style: {
+            shape: 'ellipse',
+            label: 'data(id)',
+            width: 'label',
+            backgroundColor: NrwlPallette.white,
+            'border-style': 'solid',
+            'border-color': NrwlPallette.black,
+            'border-width': '1px',
+            'text-halign': 'center',
+            'text-valign': 'center',
+            'padding-left': '16px',
+          },
+        },
+        {
+          selector: 'node[type="e2e"]',
+          style: {
+            shape: 'ellipse',
+            label: 'data(id)',
+            width: 'label',
+            backgroundColor: NrwlPallette.white,
+            'border-style': 'solid',
+            'border-color': NrwlPallette.black,
+            'border-width': '1px',
+            'text-halign': 'center',
+            'text-valign': 'center',
+            'padding-left': '16px',
+          },
+        },
+        {
+          selector: 'node.focused',
+          style: {
+            color: NrwlPallette.twilight,
+            'border-color': NrwlPallette.twilight,
+          },
+        },
+        {
+          selector: 'node.affected',
+          style: {
+            'border-color': NrwlPallette.red,
+          },
+        },
+        {
+          selector: 'edge',
+          style: {
+            width: '1px',
+            'line-color': NrwlPallette.black,
+            'curve-style': 'straight',
+            'target-arrow-shape': 'triangle',
+            'target-arrow-fill': 'filled',
+            'target-arrow-color': NrwlPallette.black,
+          },
+        },
+        {
+          selector: 'edge.affected',
+          style: {
+            'line-color': NrwlPallette.red,
+            'target-arrow-color': NrwlPallette.red,
+          },
+        },
+        {
+          selector: 'edge.implicit',
+          style: {
+            label: 'implicit',
+            'font-size': '8px',
+            'edge-text-rotation': 'autorotate',
+          },
+        },
+        {
+          selector: 'edge.dynamic',
+          style: {
+            'line-dash-pattern': [5, 5],
+            'line-style': 'dashed',
+          },
+        },
+        {
+          selector: ':parent',
+          style: {
+            'background-opacity': 0.5,
+            'background-color': NrwlPallette.twilight,
+            'border-color': NrwlPallette.black,
+            label: 'data(label)',
+            'text-halign': 'center',
+            'text-valign': 'top',
+            'font-weight': 'bold',
+            'font-size': '24px',
+          },
+        },
+      ],
+    });
+
+    this.graph.anywherePanning();
+
+    this.graph.on('zoom', () => {
+      if (this.openTooltip) {
+        this.openTooltip.hide();
+        this.openTooltip = null;
+      }
+    });
+  }
+
+  render() {
```

Comment:
I imagine this as a presentation component that takes in the data that it renders.